app/views/reports/_auditing_report.html.haml

```diff
@@ -46,19 +46,25 @@
               =label_tag "title", t(:documents)[:title]+t(:actions)[:required], class: "col-sm-3 control-label"
               .col-sm-9= text_field_tag "title", title, class: "form-control"
             .form-group
-              = label_tag "start_date", t(:reports)[:start_date]+t(:actions)[:required], class: "col-sm-3 control-label"
-              .col-sm-9
-                = text_field_tag "start_date", nil, class: "datepicker form-control", readonly: true
+              = label_tag "start_date", t(:reports)[:start_date]+t(:actions)[:required], class: "col-sm-4 col-form-label"
```

Comment:
Could you go through these modals and remove the `col-*` classes? I think the styling was a bit different previously where the label was beside the input so it's not actually doing anything now. Also the `.row`s and `col-*-12`s aren't needed anymore and can be removed in these modals.

Reply:
Removing these completely messed up the form.

Reply:
Done.